ts/src/flexible-event/privacy.ts

```diff
@@ -108,26 +101,27 @@ export class Config {
         epsilon
       )
       const newFlipProb = flipProbabilityDp(numStates, newEps)
-      excessive = { infoGainMax, newEps, newFlipProb }
+      excessive = { newEps, newFlipProb }
     }
 
     return { numStates, infoGain, flipProb, excessive }
   }
 }
 
-export const DefaultConfig: Readonly<Record<SourceType, Config>> = {
-  [SourceType.navigation]: new Config(
-    /*maxEventLevelReports=*/ 3,
-    new Array(8).fill(
-      new PerTriggerDataConfig(/*numWindows=*/ 3, /*numSummaryBuckets=*/ 3)
-    )
-  ),
-  [SourceType.event]: new Config(
-    /*maxEventLevelReports=*/ 1,
-    new Array(2).fill(
-      new PerTriggerDataConfig(/*numWindows=*/ 1, /*numSummaryBuckets=*/ 1)
+export function defaultConfig(
+  sourceType: SourceType,
+  vsv: VendorSpecificValues
+): Config {
+  const maxReports = vsv.defaultEventLevelAttributionsPerSource[sourceType]
+  return new Config(
+    /*maxEventLevelReports=*/ maxReports,
+    new Array(Number(vsv.triggerDataCardinality[sourceType])).fill(
+      new PerTriggerDataConfig(
+        /*numWindows=*/ maxReports,
```

Comment:
This is kind of a hack that just happens to work. Can we either specify this in the vsv, or else add a comment that this is just a lucky coincidence?

Reply:
I agree that this is a hack, but to make sure I understand: Rather than make this a VSV, we should make it a constant (1 for event-level, 3 for navigation) based on the hardcoding in Step 1 of https://wicg.github.io/attribution-reporting-api/#obtain-default-effective-windows?

Reply:
Done.